netbox/ipam/models/l2vpn.py

```diff
@@ -67,6 +67,12 @@ def __str__(self):
     def get_absolute_url(self):
         return reverse('ipam:l2vpn', args=[self.pk])
 
+    def can_add_termination(self):
```

Comment:
We probably want to wrap this with [`@cached_property`](https://docs.python.org/3/library/functools.html#functools.cached_property) to avoid repeated database queries for each call.